core/src/main/web/app/helpers/evaluatepluginmanager.js

```diff
@@ -129,9 +129,18 @@
                   .then(function(factory) {
                     if (factory !== undefined && factory.create !== undefined) {
                       return factory.create(evaluatorSettings).then(function(ev) { deferred.resolve(ev); });
-                    } else
-                      deferred.reject("no factory for evaluator plugin");
+                    } else {
+		      // or should someone pass in a scope to use, like with a method on the evaluatorLoadQueue?
+                      $rootScope.pluginId = name;
+		      var modal = $modal.open({backdrop: true,
+					       backdropClick: true,
+					       template: JST['helpers/plugin-load-error']()});
```

Comment:
Might have to use [this syntax](https://github.com/twosigma/beaker-notebook/blob/2b354551cca27d8c42663ae0f2234eecb77abf12/core/src/main/web/app/template/opennotebook.jst.html#L40) -- I was having trouble using the other in our environment